### PR TITLE
[DO NOT MERGE] Add Plan for Jobs promo to homepage

### PR DESCRIPTION
**Pull request by chao-xian**
## Trello card
https://trello.com/c/ljAYbjQc

### Before
www gov uk_(iPhone 5_SE)

### After
frontend dev gov uk_(iPhone 5_SE)

### CODE REVIEW

File: app/views/homepage/index.html.erb [hunk 1]
```diff
@@ -128,30 +128,30 @@
...
-          <a href="/government/topical-events/budget-2020" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/budget-2020--480w.jpg", {
+          <a href="/government/topical-events/a-plan-for-jobs-2020" aria-hidden="true" tabindex="-1">
+            <%= image_tag "homepage/plan-for-jobs-2020--480w.png", {
               alt: "",
               class: "home-promo__image",
               height: 400,
               loading: "lazy",
               sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
               srcset: {
```
Comment (huwd) [inline app/views/homepage/index.html.erb:138]: ! Very cool, i'd heard about srcset but not see it used.
Nice

Comment (chao-xian) [reply app/views/homepage/index.html.erb:138]: It was @injms who laid the groundwork for this!

File: app/views/homepage/index.html.erb [hunk 2]
```diff
@@ -128,30 +128,30 @@
         </div>
 
         <div class="govuk-grid-column-one-third">
-          <a href="/government/topical-events/budget-2020" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/budget-2020--480w.jpg", {
+          <a href="/government/topical-events/a-plan-for-jobs-2020" aria-hidden="true" tabindex="-1">
+            <%= image_tag "homepage/plan-for-jobs-2020--480w.png", {
               alt: "",
```
Comment (huwd) [inline app/views/homepage/index.html.erb:133]: Shouldn't get give it an alt name for accesibility?
```suggestion
              alt: "A Plan for Jobs",
```

Comment (chao-xian) [reply app/views/homepage/index.html.erb:133]: I think it's arguable that as the "image" is text repeating the linked text, adding this alt-text would mean the user gets repeated text read out.

File: app/views/homepage/index.html.erb (hunk 1 above)
Comment (huwd) [reply app/views/homepage/index.html.erb:138]: @injms legend.

File: app/views/homepage/index.html.erb (hunk 2 above)
Comment (injms) [reply app/views/homepage/index.html.erb:133]: Agreed - I think that the blank alt text is acceptable here as the suggestion is repeating the heading further down the page. Anyone using a screen reader or similar assistive tech wouldn't gain anything by having this alt text read out.

**Pull request merged**